From c255462f64529269e6576ce7d354f8fed4c27dd9 Mon Sep 17 00:00:00 2001
From: Bin Han <bin.b.han@intel.com>
Date: Thu, 22 Oct 2015 18:21:59 +0800
Subject: [PATCH] atomisp2: use shisp_merr.bin for BTNS

To avoid conflict with COHO, we choose to use shisp_merr.bin
for BTNS.

Change-Id: I386f4ffad3c1e2d8fb0e52b1904411705b0174b3
Original-Tracked-On: #H1504075359
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-7449
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/81053
Approver: Tian Shu Qiu <tian.shu.qiu@intel.com>
Reviewed-by: GuoqingX Zhang <guoqingx.zhang@intel.com>
Build: Dessert M BXT RVP (SC)
Build: Dessert M COHO (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
---
 .../camera/drivers/media/pci/atomisp2/css2400b0_v21_build/Makefile    | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/Makefile b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/Makefile
index e8f91af..cf37a04 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/Makefile
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/Makefile
@@ -2,4 +2,8 @@ ccflags-y += -DATOMISP_POSTFIX=\"css2400b0_v21\" -DISP2400B0
 ISP2400B0 := y
 postfix := css2400b0_v21
 
+ifeq ($(CONFIG_PLATFORM_BTNS),y)
+ccflags-y += -DATOMISP_FWNAME=\"shisp_merr.bin\"
+endif
+
 include $(srctree)/$(src)/../Makefile.common
-- 
1.9.1

